src/keycloak/chart/values.schema.json

```diff
@@ -0,0 +1,384 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "affinity": {
+      "type": "string"
+    },
+    "autoscaling": {
+      "type": "object",
+      "properties": {
+        "behavior": {
+          "type": "object",
+          "properties": {
+            "scaleDown": {
```

Comment:
Somewhat relevant to previous comment, this object for `behavior` is dropped directly into the [HPA template](https://github.com/defenseunicorns/uds-core/blob/main/src/keycloak/chart/templates/hpa.yaml#L22). Behavior in [that spec](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#configurable-scaling-behavior) actually supports a few more fields, notably `scaleUp` is missing here. This doesn't cause any immediate issues because we allow additionalProperties but might be good to update this one to add `scaleUp` at least.

As a sidenote - it would be nice to try and pull in the upstream k8s resource definitions/schema for things like this so that we aren't reinventing the wheel.